bridge/github/config.go

```diff
@@ -205,13 +206,61 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("Detected Projects:")
+
+		repos, nil := repo.ListRemotes()
```

Comment:
You need to check for error here. `nil` is a special key word in Go, it would be better to do something like
```go
repos, err := repo.ListRemotes()
if err != nil {
    return "", "", err
}
```